client/src/components/BookDetail.js

```diff
@@ -0,0 +1,139 @@
+import PropTypes from 'prop-types'
+import RectangleButton from './RectangleButton'
+import styled from 'styled-components/macro'
+
+/** @component */
```

Comment:
I don't think you need this `/** @component */`, also in your other files.

Reply:
I thought i need it [to work in styleguidist with styled-components](https://github.com/styleguidist/react-styleguidist/blob/master/docs/Thirdparties.md#styled-components). 🤔